pkg/controller/che/che_controller.go

```diff
@@ -269,6 +270,9 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 		// Che cannot be deployed with current configuration.
 		// Print error message in logs and wait until the configuration is changed.
 		logrus.Error(err)
+		if err := r.SetStatusDetails(instance, request, failedValidationReason, err.Error(), ""); err != nil {
```

Comment:
what is this good for and how is it related to single-host ?

Reply:
As the comment says:
```
		// Che cannot be deployed with current configuration.
		// Print error message in logs and wait until the configuration is changed.
```